test/TestProjects/SingleTopLevelClientWithoutOperations-LowLevel/SingleTopLevelClientWithoutOperations.json

```diff
@@ -11,9 +11,9 @@
   "produces": [ "application/json" ],
   "consumes": [ "application/json" ],
   "paths": {
-    "/client1": {
+    "/client5": {
```

Comment:
Just curious why we're renumbering here?

Reply:
To avoid conflict with another test project (otherwise test compiles but fails later with non-obvious error).